electron-cash

```diff
@@ -383,14 +440,14 @@ def run_start(config, config_options, subcommand):
         daemon_thread = daemon.Daemon(config, file_desc, False, plugins)
         daemon_thread.start()
         if config.get("websocket_server"):
-            from electroncash import websockets  # pylint: disable=C0415
```

Comment:
This needs to be reverted back to a lazy import here. See previous comment.

Reply:
Thank you for catching this.  I reverted this line and added a comment explaining why the lazy import is needed.